Support/using-the-help-desk-web-ui.md

```diff
@@ -20,10 +20,12 @@
 </ul> 
 </ul> 
 <h3>Additional Resources</h3> 
+<p><a href="https://www.ctl.io/knowledge-base/support/zendesk-login-help-for-helpdesk-ticketing-and-kb-access/">Accessing the Support ticketing web interface</a>
```

Comment:
Please rewrite this using markup. The reason is so that we can check that referred KB articles exist before merging PRs. The referenced KB articles should be referenced as file paths relative to the article. E.g., `[Ticket Prioritization Matrix](ticket-prioritization-matrix.md)`